components/character-card.js

```diff
@@ -1,32 +1,64 @@
 import { createElement } from "../lib/elements";
 import styles from "./characterCard.module.css";
 
-export default function createCharacterCard({ name, location, appearance }) {
+export default function createCharacterCard({
+  image,
+  name,
+  status,
+  species,
+  location,
+  episode,
+}) {
+  const firstEpisode = episode[0];
+  const locationUrl = location.url;
+
+  const firstEpisodeElement = createElement(
+    "span",
+    { className: styles.cardAppearance },
+    [firstEpisode]
```

Comment:
You don't need the `[ ]` here

```suggestion
    firstEpisode
```